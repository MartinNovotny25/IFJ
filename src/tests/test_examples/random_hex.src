foo('EC085AE936CE45BCD476',
'420D80X0561D69625XXX',
'DD2D88AD0CX\\466X3297'
'DDD\\3\x974X7C61A56BEX'
'3X28C9106AB993AD96X8'
'D14A41\\8CXB4370C\\E7X'
'X0E65E4\x705X166X\\0\\C'
'X63ED5X82A24CAE5B07E'
'41745B\x79302\\9X9\\17X',
'\\C78775C100A5X\\95X35',
'E404B88\\B20X65\x462X6',
'6296E0XA8632B8EE\\8C9',
'6XCDA535\xC0823X513B2'
'\\\x7A\X7A\x68\Z\zz')
