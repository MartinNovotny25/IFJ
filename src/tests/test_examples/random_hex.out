TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, EC085AE936CE45BCD476)
TOKEN(,)
TOKEN(EOL)
TOKEN(STRING, 420D80X0561D69625XXX)
TOKEN(,)
TOKEN(EOL)
TOKEN(STRING, DD2D88AD0CX\466X3297)
TOKEN(EOL)
TOKEN(STRING, DDD\3�4X7C61A56BEX)
TOKEN(EOL)
TOKEN(STRING, 3X28C9106AB993AD96X8)
TOKEN(EOL)
TOKEN(STRING, D14A41\8CXB4370C\E7X)
TOKEN(EOL)
TOKEN(STRING, X0E65E4p5X166X\0\C)
TOKEN(EOL)
TOKEN(STRING, X63ED5X82A24CAE5B07E)
TOKEN(EOL)
TOKEN(STRING, 41745By302\9X9\17X)
TOKEN(,)
TOKEN(EOL)
TOKEN(STRING, \C78775C100A5X\95X35)
TOKEN(,)
TOKEN(EOL)
TOKEN(STRING, E404B88\B20X65F2X6)
TOKEN(,)
TOKEN(EOL)
TOKEN(STRING, 6296E0XA8632B8EE\8C9)
TOKEN(,)
TOKEN(EOL)
TOKEN(STRING, 6XCDA535�823X513B2)
TOKEN(EOL)
TOKEN(STRING, \z\X7Ah\Z\zz)
TOKEN())
TOKEN(EOL)
TOKEN(EOF)
